src/templates/ja/2019/base.html

```diff
@@ -30,103 +30,103 @@
 {% block part %}部{% endblock %}
 {% block chapter %}章{% endblock %}
 
-{% block start_exploring %}Start exploring{% endblock %}
+{% block start_exploring %}探索を開始{% endblock %}
 
-{% block menu_title %}Page menu{% endblock %}
-{% block open_the_menu %}Open the menu{% endblock %}
-{% block close_the_menu %}Close the menu{% endblock %}
+{% block menu_title %}ページメニュー{% endblock %}
+{% block open_the_menu %}メニューを開く{% endblock %}
+{% block close_the_menu %}メニューを閉じる{% endblock %}
 
-{% block index_nav_title %}Chapter table of contents{% endblock %}
-{% block index_title %}Index{% endblock %}
-{% block open_the_index %}Open the Table of Contents{% endblock %}
-{% block close_the_index %}Close the Table of Contents{% endblock %}
+{% block index_nav_title %}章の目次{% endblock %}
+{% block index_title %}インデックス{% endblock %}
+{% block open_the_index %}目次を開く{% endblock %}
+{% block close_the_index %}目次を閉じる{% endblock %}
 
-{% block open %}Open{% endblock %}
-{% block close %}Close{% endblock %}
+{% block open %}開く{% endblock %}
+{% block close %}閉じる{% endblock %}
 
 {% macro onTwitter(twitterHandle) %}@{{twitterHandle}} on Twitter{% endmacro %}
 {% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} on GitHub{% endmacro %}
-{% macro website(authorName) %}{{authorName}} website{% endmacro %}
+{% macro website(authorName) %}{{authorName}} ウェブサイト{% endmacro %}
 
-{% macro edition(year) %}{{ year }} Edition{% endmacro %}
+{% macro edition(year) %}{{ year }} 版{% endmacro %}
 
-{% block footer_title %}Footer navigation{% endblock %}
-{% block copyright %}© 2019 Web Almanac. Licensed under <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/LICENSE">Apache 2.0</a>.{% endblock %}
-{% block accessibility_statement %}Accessibility Statement{% endblock %}
+{% block footer_title %}フッターナビゲーション{% endblock %}
+{% block copyright %}© 2019 Web Almanac. Licensed under <a href=\"https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/LICENSE">Apache 2.0</a>.{% endblock %}
```

Comment:
Missed a translation here?

Again using Google Translate this might be this?
```suggestion
{% block copyright %}
©2019ウェブ年鑑。  <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/LICENSE">Apache 2.0</a>の下でライセンスされています。{% endblock %}
```

Reply:
I think License sentence don't need translate.
I see it a lot, english license sentence of japanese web page.

Reply:
OK